demo/src/App.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
```

Comment:
Please be consistent with curly braces `{` `}` across this file. This applies to both JS and JSX.